torchci/lib/drciUtils.ts

```diff
@@ -510,12 +510,11 @@ export async function getPRMergeCommits(
   });
 
   // If the array is empty, the PR hasn't been merged yet
-  return results.reduce((acc: { [prNumber: number]: string[] }, row: any) => {
-    if (!acc[row.pr_num]) {
-      acc[row.pr_num] = [];
+  return results.reduce((acc: Map<number, string[]>, row: any) => {
+    if (!acc.has(row.pr_num)) {
+      acc.set(row.pr_num, []);
     }
-
-    acc[row.pr_num].push(row.merge_commit_sha);
+    acc.get(row.pr_num)!.push(row.merge_commit_sha);
```

Comment:
idk what happened to the typing here, it starts out as a map, but then acc is {key: string} and the return value is also a map

so im just making it always a map